features/api_token_settings.feature

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Shall we have single purpose PRs?
How does api token check correlate with "Filter people with alphabet"?

Reply:
done